belgif-rest-problem-it/belgif-rest-problem-spring-boot-3-it/src/test/java/io/github/belgif/rest/problem/RestProblemSpringBoot3IT.java

```diff
@@ -26,10 +28,14 @@ protected Stream<String> getClients() {
         return Arrays.stream(Client.values()).map(Client::name);
     }
 
+    // On Spring Boot 3.x only (for now), a proper resourceNotFound problem is returned
     @Override
-    @Disabled("Throws org.springframework.web.servlet.resource.NoResourceFoundException on latest Spring Boot 3")
+    @Test
     void notFound() {
-        super.notFound();
+        getSpec().when().get("/not/found").then().assertThat()
+                .statusCode(404)
+                .body("type", equalTo("urn:problem-type:belgif:resourceNotFound"))
+                .body("detail", equalTo("No resource frontend/not/found found"));
```

Comment:
Interesting that the leading `/` is removed by Spring, should we maybe add it if the path doesn't start with one? Examples in Belgif guide have a leading `/` for their resource path.

Otherwise we should probably update our unit test as it currently doesn't reflect the actual exception thrown by Spring.